src/ResultWriter/faultoutput.f90

```diff
@@ -766,7 +766,7 @@ END SUBROUTINE calc_FaultOutput
   SUBROUTINE write_FaultOutput_elementwise(EQN, DISC, MESH, IO, MPI, MaterialVal, BND, time, dt)
       !-------------------------------------------------------------------------!
       USE JacobiNormal_mod
-      use FaultWriter
+      ! use FaultWriter ! DEPRECATED: can be removed
```

Comment:
do not commet but remove?

Reply:
it will be removed later. I am keeping this in my mind